en/config/safety.md

```diff
@@ -0,0 +1,225 @@
+This section shows how to set the land/loiter behaviour after returning.
+
+![Safety - Return Home Settings (QGC)](../../images/qgc/setup/safety_return_home.png)
+
+The settings and underlying parameters are shown below:
+
+Setting | Parameter | Description
+--- | --- | ---
+Climb to altitude | [RTL_RETURN_ALT](../advanced_config/parameter_reference.md#RTL_RETURN_ALT) | Vehicle ascend to this minimum height (if below it) for the return flight.
+Return behaviour |  | Choice list of *Return then*: Land, Loiter and do not land, or Loiter and land after a specified time.
+Loiter Altitude | [RTL_DESCEND_ALT](../advanced_config/parameter_reference.md#RTL_DESCEND_ALT) | If return with loiter is selected you can also specify the altitude at which the vehicle hold.
+Loiter Time | [RTL_LAND_DELAY](../advanced_config/parameter_reference.md#RTL_LAND_DELAY) | If return with loiter then land is selected you can also specify how long the vehicle will hold.
+
+> **Note** The return behavour is defined by [RTL_LAND_DELAY](../advanced_config/parameter_reference.md#RTL_LAND_DELAY). If negative the vehicle will land immediately. Additional information can be found in [Return mode](../flight_modes/return.md).
+
+
+### Land Mode Settings
+
+*Land at the current position* is a common [failsafe action](#failsafe_actions) that engages [Land Mode](../flight_modes/land.md). 
+This section shows how to set whether the vehicle will automatically disarm after landing. For Multicopters (only) you can additionally set the descent rate.
+
+![Safety - Land Mode Settings (QGC)](../../images/qgc/setup/safety_land_mode.png)
+
+The settings and underlying parameters are shown below:
+
+Setting | Parameter | Description
+--- | --- | ---
+Disarm After | [COM_DISARM_LAND](../advanced_config/parameter_reference.md#COM_DISARM_LAND) | Select checkbox to specify that the vehicle will disarm after landing, and enter delay after landing before disarming (must be non-zero). 
+Landing Descent Rate | [MPC_LAND_SPEED](../advanced_config/parameter_reference.md#MPC_LAND_SPEED) | Rate of descent (MC only).
+
+
+## Other Failsafe Settings {#failsafe_other}
+
+This section contains information about failsafe settings that cannot be configured through the *QGroundControl* [Safety Setup](#qgc_safety_setup) page.
+
+### Position (GPS) Loss Failsafe
+
+The Position Loss failsafe is triggered if the quality of the PX4 position estimate falls below acceptable levels (this might be caused by GPS loss).
+
+The relevant parameters are shown below:
+
+Parameter | Description
+--- | ---
+[COM_POS_FS_DELAY](../advanced_config/parameter_reference.md#COM_POS_FS_DELAY) | Delay after loss of position before the failsafe is triggered.
+[CBRK_GPSFAIL](../advanced_config/parameter_reference.md#CBRK_GPSFAIL) | Circuit breaker that can be used to disable GPS failure detection.
```

Comment:
Is this correct - ie should it be mentioned here. Is it really GPS or it kills all of position estimation failure warnings?

Reply:
I noticed different failsafe action in Position mode than in Mission mode for GPS failure, as described [here](https://github.com/PX4/Firmware/issues/9336). The description you wrote holds for "Global position loss" failsafe which is triggered upon GPS failure in mission mode. However in position mode, the GPS (or any other position source) failsafe action is defined by  COM_POSCTL_NAVL.

Reply:
Maybe it is worth separating "Global Position Loss Failsafe" from "Local Position Loss Failsafe"

Reply:
> for a time while trying to regain GPS, and then enter flight termination.

When I simulated GPS fail with multicopter in mission mode, it "landed on spot" rather than "terminating flight". I learned from your other comments that "terminating flight" could mean kill switch behaviour/disarming motors immediately so it is better to confirm and distinguish between the two.

Reply:
Thanks @barzanisar . I have re-written this section. I think that behaviour is somewhere between what you and I wrote - essentially COM_POSCTL_NAVL applies everywhere. 

AFAIK little evidence there is a difference between local and global position handling, so not worth differentiating. From what Paul has told me the source is always EKF.